cli/src/commit_hand.rs

```diff
@@ -0,0 +1,27 @@
+use crate::accounts::{ commit_hand };
+use aze_lib::constants::{ PLAYER_CARD1_SLOT, PLAYER_CARD2_SLOT };
+use clap::Parser;
+use miden_objects::{ 
+    accounts::AccountId,
+    Felt, FieldElement
+};
+
+#[derive(Debug, Clone, Parser)]
+pub struct CommitHandCmd {
```

Comment:
I think we are caching the `game_id` and `player_id` in some config file ? We can read it from there as well and if we are not caching let's cache it.